index/index.go

```diff
@@ -875,6 +875,18 @@ func (r *Reader) LabelValues(names ...string) (StringTuples, error) {
 	return st, nil
 }
 
+func (r *Reader) LabelNames() []string {
+	const sep = "\xff"
```

Comment:
this cons is used in many other places , maybe a good follow up PR would be to make ti global.
It is already in the labels package to maybe can just export is and use it everywhere  as labels.Separator